deployment/ansible/group_vars/all

```diff
@@ -30,6 +30,7 @@ postgis_package_version: "2.3.1+dfsg-1.pgdg14.04+1"
 # azavea.docker
 # 1.11.* to match AWS ECS
 docker_version: 1.11.*
+docker_options: "--storage-opt dm.basesize=20G"
```

Comment:
There really isn't any explicit documentation, but that's my takeaway from the references to bind-mounts in https://docs.docker.com/engine/tutorials/dockervolumes/#mount-a-host-directory-as-a-data-volume.

```bash
ubuntu@ip-172-31-52-80:~$ mkdir joker
ubuntu@ip-172-31-52-80:~$ docker run -ti --rm -v ${PWD}/joker:/joker -w /joker debian:jessie /bin/bash
Unable to find image 'debian:jessie' locally
jessie: Pulling from library/debian
693502eb7dfb: Already exists
Digest: sha256:52af198afd8c264f1035206ca66a5c48e602afb32dc912ebf9e9478134601ec4
Status: Downloaded newer image for debian:jessie
root@2b2bbba847ad:/joker# ls
root@2b2bbba847ad:/joker# fallocate -l 20G joker.img
root@2b2bbba847ad:/joker# ls -lh
total 21G
-rw-r--r-- 1 root root 20G Mar  8 14:58 joker.img
root@2b2bbba847ad:/joker# exit
exit
ubuntu@ip-172-31-52-80:~$ ls -lh joker/
total 21G
-rw-r--r-- 1 root root 20G Mar  8 09:58 joker.img
ubuntu@ip-172-31-52-80:~$ docker info | grep Storage
 WARNING: Usage of loopback devices is strongly discouraged for production use. Use `--storage-opt dm.thinpooldev` to specify a custom block storage device.
WARNING: No swap limit support
Storage Driver: devicemapper
```